# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/android/AccessibleWrap.h
# Commit: 16b49e8c315b
# Full Hash: 16b49e8c315bb512c0bc406670d3b4be653185b4
# Author: James Teh <jteh@mozilla.com>
# Date: 2023-11-30 04:12:04
# Regressor Bug: 1862802
# File Overlap Count: 1
# Description:
#   Bug 1862802 part 5: Remove virtual cursor change event. r=eeejay
#   
#   Depends on D192645
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D192646
# ==============================================================================

diff -r 5a6669e44c53 -r 16b49e8c315b accessible/android/AccessibleWrap.h
--- a/accessible/android/AccessibleWrap.h	Wed Nov 29 03:53:20 2023 +0000
+++ b/accessible/android/AccessibleWrap.h	Wed Nov 29 03:53:20 2023 +0000
@@ -27,9 +27,6 @@
 
   virtual bool DoAction(uint8_t aIndex) const override;
 
-  MOZ_CAN_RUN_SCRIPT_BOUNDARY
-  virtual bool PivotTo(int32_t aGranularity, bool aForward, bool aInclusive);
-
   static Accessible* ExploreByTouch(Accessible* aAccessible, float aX,
                                     float aY);
 